xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -1146,6 +1146,13 @@ int CDVDVideoCodecFFmpeg::FilterOpen(const std::string& filters, bool scale)
     return result;
   }
 
+  char* graphDump = avfilter_graph_dump(m_pFilterGraph, nullptr);
+  if (graphDump)
```

Comment:
@pkerling this should only be dumped if component logging for video is turned on. the filter my get drained before ffmpeg and in this case it gets reopened a couple of times before eof. -> results in log spam